src/test/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionsControllerIntegrationTest.kt

```diff
@@ -41,7 +38,7 @@ class IdentificationSessionsControllerIntegrationTest(
         webTestClient
             .post()
             .uri(URI.create("http://localhost:$port/api/v1/identification/sessions"))
-            .headers { headers -> headers.setBasicAuth(username, password) }
+            .headers { headers -> setAuthorizationHeader(headers) }
```

Comment:
Maybe we can have this shorter? I think the method name makes it clear, that the value are headers, so we could use `.headers { setAuthorizationHeader(it) }`? But I'm fine with the a bit more verbose solution, too.

Reply:
Good catch. Done.